build/ci.go

```diff
@@ -390,6 +472,58 @@ func downloadLinter(cachedir string) string {
 	return filepath.Join(cachedir, base, "golangci-lint")
 }
 
+// getProtocArchiveBaseName returns the name of the protoc archive file for
+// the current system, stripped of version and file suffix.
+func getProtocArchiveBaseName() (string, error) {
```

Comment:
Please note that as per usual conventions, you can just call this function `protocArchiveBaseName` without the `get`.